frontend/src/components/Chessboard.vue

```diff
@@ -198,6 +202,19 @@ export default {
         fen: historyGame.fen(),
         lastMove: lastMove
       });
+      if (ply != this.game.history().length) {
+        this.board.set({
+          movable: {
+            color: null
+          }
+        });
+      } else {
+        this.board.set({
```

Comment:
This piece of code doesn't work as supposed. Please remove it.

Reply:
Removed